### PR TITLE
Avoid MegaLinter false positive with .secretlintignore

### PR DESCRIPTION
**Pull request by nvuillam**
Not sure it will work the first time, if necessary I'll patch MegaLinter :)

### CODE REVIEW

**Comment by onedr0p** [comment]
https://github.com/oxsecurity/megalinter/issues/816#issuecomment-939210391

Seems like this can point to the .gitignore instead?

**Comment by nvuillam** [comment]
@onedr0p with MegaLinter v6.3.0, I added the [capability to refer to a .secretlintignore file](https://oxsecurity.github.io/megalinter/latest/descriptors/repository_secretlint/) in .github/linters, please can you allow the job to run to verify that is now ok ?

It will avoid you the MegaLinter failing jobs because of the example key in README file :)

**Comment by onedr0p** [comment]
I ran it, does that seem expected?

**Comment by nvuillam** [comment]
Nope, it find the .secretlintignore and uses it but does not take in account the README.md in it :/
Maybe secretlint is not happy with folder stuff... i try something else

**Comment by onedr0p** [comment]
Maybe test this out on a repo you own so we don't have to go back and forth with running the CI 😄

**Comment by nvuillam** [comment]
> Maybe test this out on a repo you own so we don't have to go back and forth with running the CI 😄

Indeed :)

Meanwhile if you're tired to the false positive, you can define variable `REPOSITORY_SECRETLINT_DISABLE_ERRORS: true` :)

**Comment by fabricesemti80** [comment]
I think this still did not work...

```
❌ Linted [REPOSITORY] files with [secretlint]: Found 1 error(s) - (1.2s) (expand for details)
  - Using [secretlint v5.2.3] https://oxsecurity.github.io/megalinter/latest/v6.4.0/descriptors/repository_secretlint
  - MegaLinter key: [REPOSITORY_SECRETLINT]
  - Rules config: [.secretlintrc.json]
  - Ignore file: [/github/workspace/.github/linters/.secretlintignore]
  --Error detail:
  
  /github/workspace/README.md
    483:14  error  [PrivateKey] found private key: -----BEGIN OPENSSH PRIVATE KEY-----
                    ...
                -----END OPENSSH PRIVATE KEY-----  @secretlint/secretlint-rule-preset-recommend > @secretlint/secretlint-rule-privatekey
  
  ✖ 1 problem (1 error, 0 warnings)
  ```
 At least I still see this in the output...?

**Comment by nvuillam** [comment]
@fabricesemti80 it's an issue with secretlint >_<
It will find another way

**Comment by onedr0p** [comment]
For that file, just remove it from your own repos README.md

**Comment by onedr0p** [comment]
Ideally we should use `secretlintignore` and ignore the README

**Comment by nvuillam** [comment]
I'm sure it will work like that but I didn't want to bother your root folder with a .secretlintignore :/
If it is ok for you I can do that and the PR will pass :)

**Comment by onedr0p** [comment]
That should be be fine.

File: README.md [hunk 1]
```diff
@@ -470,6 +470,7 @@ The benefits of a public repository include:
       ```
   2. Paste public key in the deploy keys section of your repository settings
   3. Create sops secret in `cluster/flux/flux-system/github-deploy-key.sops.yaml` with the contents of:
+      <!-- secretlint-disable -->
```
Comment (onedr0p) [inline README.md:473]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -489,6 +490,7 @@ The benefits of a public repository include:
               github.com ecdsa-sha2-nistp256 ...
               github.com ssh-rsa ...
       ```
+      <!-- secretlint-enable -->
```
Comment (onedr0p) [inline README.md:493]:
```suggestion
```

**Review by onedr0p — CHANGES_REQUESTED** [review]
No need for this in the readme

File: .secretlintignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+megalinter-reports
+README.md
```
Comment (onedr0p) [inline .secretlintignore:2]:
```suggestion
README.md

```

**Review by onedr0p — APPROVED** [review]

**Pull request merged**